README.md

```diff
@@ -1,168 +1,222 @@
 # DSP-UI library
 
-This is the demo and developing environment for DSP-UI library (@knora/ui) composed of 4 modules.
+This is the demo and developing environment for DSP-UI-LIB library (@dasch-swiss/dsp-ui) composed of 4 modules.
 
-The modules helps to create a graphical user interface, a web application to use [DSP-API](https://www.knora.org) in a quick and simple way. The modules are written in typescript to use them with [Angular](https://angular.io) (version 8). We decided to style the components and directives with [Angular Material design](https://material.angular.io).
+The modules help to create a graphical user interface, a web application to use [DSP-API](https://www.knora.org) in a quick and simple way. The modules are written in Typescript to use them with [Angular](https://angular.io) (version 9). We decided to style components and directives with [Angular Material design](https://material.angular.io).
 
-DSP-UI library implements [DSP-API-JS-LIB](https://www.npmjs.com/package/@knora/api) to connect with the Knora web API. DSP-API is a software framework for storing, sharing, and working with primary sources and data in the humanities.
+DSP-UI-LIB implements [DSP-JS-LIB](https://www.npmjs.com/package/@knora/api) to connect with the DSP API. DSP-API is a software framework for storing, sharing, and working with primary sources and data in the humanities.
```

Comment:
implements -> make use of
insert a link to https://www.knora.org

Reply:
I think it is enough to describe the license of the UI lib here.

Reply:
> insert a link to https://www.knora.org

or to the documentation on docs.dasch.swiss?

Reply:
> > insert a link to https://www.knora.org
> 
> or to the documentation on docs.dasch.swiss?

that would be fine

Reply:
https://github.com/dasch-swiss/knora-ui-ng-lib/pull/72/commits/52cbed75b055c21f36191e038fb410230344e7df